app/register/template.hbs

```diff
@@ -41,10 +41,11 @@
             <label class='form-label' for='password'>Password</label>
           </div>
           <div class='text-center text-lg-start mt-4 pt-2'>
-            <button
-              type='submit'
-              class='btn btn-primary btn-lg button-login'
-            >Register new user</button>
+            <Shared::Button
+              class='btn-primary btn-lg button-login'
```

Comment:
Maybe it's time to add additional property to your Shared::Button like `@size="lg"` (or "sm") which will inject this extra class?